easybuild/easyconfigs/a/ASAP3/ASAP3-3.13.2-foss-2023a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.13.2'
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
I suspect this can go to  the `gfbf` toolchain, no?

Reply:
No, ASAP uses MPI communication.